### PR TITLE
Add WA state and MA programs

### LINKED ISSUE #810: Add Washington as a state
Washington's only tax code in effect is the Working Families Tax Credit, which is now in OpenFisca US. Their new capital gains tax is held up in court, and only affects households that won't be in the CPS, and we don't have capital gains inputs in the household page. Their SSP is also effectively closed. So I think we can add the state to PolicyEngine US: 
- [ ] Add WFTC parameters to the Policy page
- [ ] Add WA to the drop-down with US and MA

### LINKED ISSUE #822: Age distribution chart doesn't work in US
Works fine in UK, but e.g. from https://policyengine.org/us/population-impact?gov_irs_income_bracket_rates_1=14
image

https://github.com/PolicyEngine/openfisca-us/issues/1062 should fix this. If not I'd suggest disabling it for now if there's a bigger issue around the age variable.

### PR DESCRIPTION
**Pull request by MaxGhenis**
Fixes #810 by adding WA's capital gains tax and Working Families Tax Credit, and as a population impact drop-down item

Also adds more MA programs recently implemented in OpenFisca US

### CODE REVIEW

File: policyengine/countries/us/reform_parameters.yaml [hunk 1]
```diff
@@ -140,6 +140,26 @@ reforms:
...
+        tests:
+          - increases_net_income: false
+    abolish_wa_wftc:
+      values:
+        2010-01-01: false
+      metadata:
+        label: Abolish WA Working Families Tax Credit
+        name: abolish_wa_wftc
+        unit: abolition
+        variable: wa_wftc_max_amount
```
Comment (MaxGhenis) [inline policyengine/countries/us/reform_parameters.yaml:160]: Does this work, given [`wa_wftc_max_amount` is a scale parameter](https://github.com/PolicyEngine/openfisca-us/blob/master/openfisca_us/parameters/gov/states/wa/tax/income/credits/working_families_tax_credit/amount.yaml)?

Comment (MaxGhenis) [reply policyengine/countries/us/reform_parameters.yaml:160]: I should probably point it to a variable instead of a parameter then!

**Comment by MaxGhenis** [comment]
The standard and itemized deduction submenus break for some reason. `aged_blind_std_age` works, but [`standard_deduction`](https://github.com/PolicyEngine/openfisca-us/blob/master/openfisca_us/parameters/gov/irs/deductions/standard/amount.yaml) and `aged_blind_std`, both of which are indexed on `filing_status`, break it. I haven't looked more into the itemized menu items. They appear to be correctly specified, so I'm not sure why it's breaking.

The WA and MA programs work though, I just want to dig a bit more into the results from the new policies before merging. Here are simulations of repealing them.

## WA Working Families Tax Credit

Expected top-line results.
image

But strange top 1% share result. I'm not sure if low N from filtering to WA can entirely explain this.
image

## WA capital gains tax

The budgetary impact is surely distorted by the CPS's truncation, given it has a $250k standard deduction.
image

Increases inequality as expected:
image

## MA dependent care credit

I think this has a small impact by itself because filers can only either claim it or the dependent credit, so the dependent credit fills in the gap of its repeal.
image

**Comment by nikhilwoodruff** [comment]
OK, so the itemized page breaks because the SALT cap parameter doesn't have a breakdown specified.

**Comment by nikhilwoodruff** [comment]
The standard deduction page is breaking because the standard deduction parameter isn't named by metadata.

**Comment by MaxGhenis** [comment]
>The standard deduction page is breaking because the standard deduction parameter isn't named by metadata.

I'm still missing what's wrong. [Here's what we have](https://github.com/PolicyEngine/openfisca-us/blob/7516982bac4a541f02ba7e806b3d947974131c08/openfisca_us/parameters/gov/irs/deductions/standard/amount.yaml#L32):
```python
metadata:
  unit: currency-USD
  label: Standard deduction
  name: standard_deduction
  period: year
  breakdown:
    - filing_status
```

**Comment by MaxGhenis** [comment]
* Bumped up the time estimate from 10 to 20 seconds based on a time test of the live site.
* Removed the deductions until we can figure out why they're breaking.
* Removed #822 as bumping OpenFisca US didn't fix the age chart locally
image

**Review by nikhilwoodruff — APPROVED** [review]

**Pull request merged**